File: Data Structures & Algorithms/binary-search/submission-0.go
func search(nums []int, target int) int {
r := len(nums)-1
l:= 0

for l<=r{
    mid := int(uint(l+r)>>1)
    if nums[mid]==target{
        return mid
    }else if nums[mid]>target{
        r = mid-1
    }else{
        l = mid+1
    }
}

return -1


}

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        answer = []
        for i in range(len(nums)):
            for j in range(len(nums)):
                if i != j:
                    if target == nums[i]+nums[j]:
                        answer.append(i)
                        answer.append(j)
                        break
        nums = set(answer)
        return list(nums)
        